src/stores/wallet.ts

```
@@ -0,0 +1,202 @@
+  // isHardware: boolean
+  // ledgerBLE: boolean
+  // ledgerApp: Record<PropertyKey, unknown> // ?
+  // instance: null | Wallet
+  // isOfflineApp: boolean
+  // web3: Record<PropertyKey, unknown> // ?
+  // ensDomains: null | string[] // ?
+  // tokens: unknown[] // ?
+  // loadingWalletInfo: boolean
+  // swapRates: unknown[]
+  // nickname: null | string
+}
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)